<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaEmergente</class>
 <widget class="QMainWindow" name="ventanaEmergente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Resultados</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.btnOk {
	background: #d30000;
    border-radius: 20px;
    border: nonepx;
    color: white;
}
.btnOk:hover{
	background-color: rgb(200, 0, 0);
}
.btnOk:pressed {
	background-color: rgb(150, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>300</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_mensaje">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>511</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Cooper Black&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_imagen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOk">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>250</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cooper Black</family>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="class" stdset="0">
      <string>btnOk {
	background: #d30000;
    border-radius: 20px;
    border: nonepx;
    color: white;
}
btnOk:hover{
	background-color: rgb(200, 0, 0);
}
btnOk:pressed {
	background-color: rgb(150, 0, 0);
}</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
